x/agent.py

```diff
@@ -38,14 +38,21 @@ class DiscreteAgent(Agent):
 
     """
     def __init__(self, model, memory, epsilon=None):
-        super(DiscretAgent, self).__init__(model, memory)
+        super(DiscreteAgent, self).__init__(model, memory)
         if epsilon is None:
             self.epsilon = lambda *args: .1
 
-    def compile(self, optimizer="sgd", loss="mse", policy_rule="max",
-                experience=None):
-        self.model.compile(optimizer, loss, policy_rule)
+    def compile(self, *args, **kwargs):
```

Comment:
this is not good, why did you take the names out? It will be difficult for somebody reading the code to understand it